pydantic_ai_slim/pydantic_ai/profiles/qwen.py

```diff
@@ -1,11 +1,30 @@
 from __future__ import annotations as _annotations
 
+from ..profiles.openai import OpenAIModelProfile
 from . import InlineDefsJsonSchemaTransformer, ModelProfile
+from ._json_schema import JsonSchema
 
 
 def qwen_model_profile(model_name: str) -> ModelProfile | None:
     """Get the model profile for a Qwen model."""
+    if model_name.startswith('qwen-3-coder'):
+        return OpenAIModelProfile(
+            openai_supports_tool_choice_required=False,
+            json_schema_transformer=NotStrictCompatibleJsonSchemaTransformer,
```

Comment:
Do we need this? There's already `openai_supports_strict_tool_definition` that we can set to `False` so we never pass `strct`.

Reply:
Right, thanks!